src/prevector.h

```diff
@@ -392,7 +392,10 @@ class prevector {
             change_capacity(new_size + (new_size >> 1));
         }
         T* ptr = item_ptr(p);
-        memmove(ptr + count, ptr, (size() - p) * sizeof(T));
+        // Passing dst variable with explicit type instead of temporary
+        // expression calms down GCC under some circumstances.
```

Comment:
In commit "refactor: Make code more tolerant of constexpr std::arrays" (05c16ae8649ee3b6bbcdbdf0541a65ccf4477537)

What is this referring to?

Reply:
I was running into an issue with `CScript` (inheriting `prevector`) and `std::array`s, similar to here: https://gitlab.freedesktop.org/pipewire/media-session/-/issues/4

Reverted this change in the latest push as I'm no longer modifying `CScript`.